File: service/SurfWebAutofac/Model/Caches/MapMainCache.cs
namespace Model.Caches;

public class MapMainCache
{
    /// <summary>
    ///     地图ID
    /// </summary>
    public required long Id { get; set; }

    /// <summary>
    ///     地图名称
    /// </summary>
    public required string Name { get; set; }

    /// <summary>
    ///     难度
    /// </summary>
    public required string Difficulty { get; set; }

    /// <summary>
    ///     图片
    /// </summary>
    public string Img { get; set; }

    /// <summary>
    ///     地图奖励关数量
    /// </summary>
    public int BonusNumber { get; set; }

    /// <summary>
    ///     地图阶段数量
    /// </summary>
    public int StageNumber { get; set; }
}

File: service/Java/SurfWeb/SurfWeb/system/src/main/java/com/surfweb/system/caches/MapMainCache.java
package com.surfweb.system.caches;

import lombok.Data;

/**
 * 地图主线缓存
 */
@Data
public class MapMainCache {

    /**
     * 地图ID
     */
    private long id;

    /**
     * 地图名称
     */
    private String name;

    /**
     * 难度
     */
    private String difficulty;

    /**
     * 图片
     */
    private String img;

    /**
     * 地图奖励关数量
     */
    private int bonusNumber;

    /**
     * 地图阶段数量
     */
    private int stageNumber;
}
